File: python/problem021.py
def d(n):
    value = 0
    for i in range(1, n):
        if n % i == 0:
            value += i
    return value

result = 0

for a in range(2, 10001):
    if a != d(a) and a == d(d(a)):
        result += a

print(result)

File: javascript/problem021.js
function d(n){
  var value = 0;
  for(i=1; i<n; i++){
    if(n % i == 0){
      value = value + i;
    }
  }
  return value;
}

var result = 0;

for(a=2; a<=10000; a++){
  if(a != d(a) && a == d(d(a))){
    result = result + a;
  }
}

console.log(result);
